### ISSUE TITLE
[Other issue]: Instructions on how to use align colinear

### ISSUE THREAD

**Issue by cardboardcode** [open-rmf/rmf_traffic_editor#504]
### Before proceeding, is there an existing issue or discussion for this?

- [X] I have done a search for similar [issues](.?q=type%3Aissue) and [discussions](https://github.com/open-rmf/rmf/discussions).

### Description

Hi Open-RMF team, may I enquire if more instructions could be provided regarding the use of `align colinear`, `features` and `constraints` as highlighted in #372? 

![image](https://github.com/user-attachments/assets/eaac5529-f017-4f8a-9334-d3a4bd21fdfd)

![image](https://github.com/user-attachments/assets/5777348e-737d-47ac-85ab-4f5c5d8c0873)

I have searched online, going through discussions, youtube videos as well as the [ROS 2 Multi Robot Book](https://osrf.github.io/ros2multirobotbook/traffic-editor.html). However, don't seem to find any substantial details on it apart from what is covered in the aforementioned Pull Request.

Once again, thank you for maintaining this repository. Based on the instructions, I think I will make a Pull Request to include within ROS 2 Multi Robot Book.

**Comment by mxgrey** [comment]
Use click+shift to select a group of vertices and then press the `/` key to tell the traffic editor that you want them to be collinear.

I'm not sure what additional details you'd like. Is there a use case where you're having difficulty applying this, or a situation where it's not working the way you'd expect?

**Comment by cardboardcode** [comment]
Apologies. After further debugging, answered my own query.

Think I got confused with my intended use of "aligning" Layers in Traffic Editor using the `features` and `constraints` in order to determine the Transform needed between Grid Cells and RMF.

The intended target feature was `Optimize Layer Transforms` which uses `features` and `constraints`, instead of `align colinear`.

Closing.

Thank you for the quick response @mxgrey
